xmpls

navinkrishnan opened the pull request:
* Remote Service
* Messaging

danjoa reviewed (commented):
I suggest we walk through that in a meeting. 

In that we should also check whether the `incidents-app/xmpls` approach is a best practice for such exercises including a lot of additions.

navinkrishnan commented:
As per the discussion, following changes are done
- Customer key field changed to `string` in main
- Adapted csv file data for ID with String as it needed to match with BusinessPartner data in xmpls
- removed db related files from xmpls
- Moved BusinessPartner related projections to a new service named `RemoteService`

@danjoa @DanSchlachter Please review and let me know if any changes needed.
